src/pcm/AudioFormat.cxx

```diff
@@ -22,15 +22,46 @@
 
 #include <assert.h>
 #include <stdio.h>
+#include <map>
+
+/**
+ * If source is based on 44.1k get nearest valid 44.1k variant from the target sample rate.
+ * For exmple if the source is 44.1k and the target is 96k it will return 88.2k
+ */
+unsigned
+determine_selective_resample_rate(unsigned source_rate, unsigned target_rate) noexcept;
+
+unsigned
+determine_selective_resample_rate(unsigned source_rate, unsigned target_rate) noexcept
+{
+	unsigned out_sample_rate = source_rate;
+	const std::map<unsigned, unsigned> lut48to41 = {
```

Comment:
This `std::map` gets initialized on each call. This is unnecessary bloat - it's the wrong data structure here. And the name of this variable is cryptic.
Why is this even a hard-coded list?

Reply:
Context of each call is if an other stream (like as in another song) is selected.
Should I make it a switch with cases ?

Reply:
I don't understand anything in your reply. Conext, what context, and what does that have to do with my review? And how does switch/case solve the issue?